private/buf/bufformat/testdata/proto3/all/v1/all.proto

```diff
@@ -69,7 +69,9 @@ message One {
   int64 one_one = 1 [ (custom.field_option) = true];
   Two one_two = 2[
   (custom.field_option) = false,
-  (custom.field_thing_option) = { foo: 6}
+  (custom.field_thing_option) = {
```

Comment:
Is the reason this is going to a newline because of the comment? I think sans comment we'd want it on a single line, so just checking.

Reply:
Yes, because of the comment.